src/app/static/src/app/types.ts

```diff
@@ -166,9 +166,8 @@ export interface UploadFile {
 
 export interface DownloadResultsDependency {
     downloadId: string
-    downloading: boolean
```

Comment:
This caused me some confusion! Here and in the renamed methods, the outputs are actually being prepared for download.